### PR TITLE
Correct Management Service Bus solution file

### LINKED ISSUE #6744: [BUG] Management Service Bus Solution file has incorrect project references
The solution file at sdk\servicebus\Microsoft.Azure.Management.ServiceBus/Microsoft.Azure.Management.ServiceBus.sln has references to projects that do not exist (they appear to have had their names simplified at some point.

To reproduce: open the solution file in Visual Studio and observe that the projects fail to load.

### PR DESCRIPTION
**Pull request by OwenDF**
Resolves #6744

### CODE REVIEW

File: sdk/servicebus/Microsoft.Azure.Management.ServiceBus/Microsoft.Azure.Management.ServiceBus.sln [hunk 1]
```diff
@@ -3,9 +3,9 @@ Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio 15
 VisualStudioVersion = 15.0.26228.4
 MinimumVisualStudioVersion = 10.0.40219.1
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.Azure.Management.ServiceBus.Tests", "tests\Microsoft.Azure.Management.ServiceBus.Tests.csproj", "{90FE65CB-D848-4ED8-97F5-7C967102CEAF}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.Azure.Management.ServiceBus.Tests", "tests\Management.ServiceBus.Tests.csproj", "{90FE65CB-D848-4ED8-97F5-7C967102CEAF}"
 EndProject
```
Comment (erich-wang) [inline sdk/servicebus/Microsoft.Azure.Management.ServiceBus/Microsoft.Azure.Management.ServiceBus.sln]: Suggest to rename csproj files with prefix **Microsoft.Azure**

**Review by erich-wang — CHANGES_REQUESTED** [review]

File: sdk/servicebus/Microsoft.Azure.Management.ServiceBus/Microsoft.Azure.Management.ServiceBus.sln (hunk 1 above)
Comment (OwenDF) [reply sdk/servicebus/Microsoft.Azure.Management.ServiceBus/Microsoft.Azure.Management.ServiceBus.sln]: Hi Erich, judging by the failures in the build checks, I'm guessing that the build pipeline is probably looking for the csproj files by name, and that changing them is a breaking change. How should we proceed?

Comment (erich-wang) [reply sdk/servicebus/Microsoft.Azure.Management.ServiceBus/Microsoft.Azure.Management.ServiceBus.sln]: After renaming csproj files, you should update **Management.ServiceBus.csproj** to **Microsoft.Azure.Management.ServiceBus.csproj** in test csproj file as well.

Comment (OwenDF) [reply sdk/servicebus/Microsoft.Azure.Management.ServiceBus/Microsoft.Azure.Management.ServiceBus.sln]: Do you mean the project reference of the unit tests to the main project? I have done that in the 2nd commit. Let me know if you meant something else.

Comment (OwenDF) [reply sdk/servicebus/Microsoft.Azure.Management.ServiceBus/Microsoft.Azure.Management.ServiceBus.sln]: Ok, having actually pushed the changes properly, everything is passing. Sorry about that.

**Review by erich-wang — APPROVED** [review]

**Pull request merged**